_includes/shared/footer/subcript.gs

```diff
@@ -0,0 +1,20 @@
+var wbook = SpreadsheetApp.openByUrl(
+  "https://docs.google.com/spreadsheets/d/1lEtNFeTlXL4u5ZBpAFAGQRmPBdR_t8y3T2U-IJkpS90/edit#gid=0"
```

Comment:
we use [openById](https://developers.google.com/apps-script/reference/spreadsheet/spreadsheet-app#openbyidid) or [getActive
](https://developers.google.com/apps-script/reference/spreadsheet/spreadsheet-app#getactive)